csharp/ql/lib/ext/System.Data.model.yml

```diff
@@ -54,6 +54,17 @@ extensions:
       - ["System.Data", "EnumerableRowCollectionExtensions", False, "Where<TRow>", "(System.Data.EnumerableRowCollection<TRow>,System.Func<TRow,System.Boolean>)", "", "Argument[0].Element", "ReturnValue.Element", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "get_Item", "(System.String)", "", "Argument[this].Element", "ReturnValue", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "set_Item", "(System.String,System.Object)", "", "Argument[1]", "Argument[this].Element", "value", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetDataTypeName", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
```

Comment:
This API seems to return a value from a fixed set of datatypes. Do we need this model?

Reply:
No, I will introduce a neutral for this one and see if we can avoid generating it alltogether!